src/wcs2_0/pom.xml

```diff
@@ -53,6 +47,13 @@ application directory.
       <artifactId>gt-xsd-wcs</artifactId>
     </dependency>
 
+    <dependency>
```

Comment:
If the integration tests have been moved, wcs 2.0 should not depend on wcs 1.1 any longer. What's failing?